test/date_test.rb

```diff
@@ -0,0 +1,93 @@
+require_relative 'test_helper'
+
+describe "Date" do
```

Comment:
Since this file is a suite of tests for `Reservation_Date` give it the title `reservation_date_test`